src/r_things.c

```diff
@@ -348,9 +349,18 @@ void R_DrawMaskedColumn(column_t *column)
       topscreen = sprtopscreen + spryscale*top;
       bottomscreen = topscreen + spryscale*column->length;
 
-      // Here's where "sparkles" come in -- killough:
-      dc_yl = (int)((topscreen+FRACUNIT-1)>>FRACBITS); // [FG] 64-bit integer math
-      dc_yh = (int)((bottomscreen-1)>>FRACBITS); // [FG] 64-bit integer math
+      if (fuzz)
```

Comment:
Hm, while this looks certainly correct, I'd like to avoid branching and special caging as much as possible. Do you think we could somehow move this into `R_DrawFuzzColumn_block()`? Perhaps by storing the frac remainder in a variable? Or by adjusting the way that the start pixel and the column length are calculated in that function?